io/disk/htx_block_devices.py.data/htx_block_devices.yaml

```diff
@@ -0,0 +1,4 @@
+parameters:
+    block_devices: 'mpatha mpathb'
```

Comment:
Have this parameter name as `disk` since that will make sure it is in line with all other tests